UI/MultiplayerLobbyWnd.cpp

```diff
@@ -37,6 +37,14 @@ namespace {
     { return ClientUI::GetFont(ClientUI::Pts())->Height(); }
     GG::Y PlayerRowHeight()
     { return PlayerFontHeight() + 2 * PlayerRowMargin(); }
+    GG::X PlayerReadyBrowseWidth()
+    { return GG::X(ClientUI::Pts() * 11); }
+
+    const boost::shared_ptr<GG::Texture> GetReadyTexture(bool ready) {
+        if (ready)
+            return GG::GetTextureManager().GetTexture(ClientUI::ArtDir() / "icons/ready.png");
```

Comment:
GG::GetTextureManager seems to not be publically accessible to my linker. Replace this with standard in FreeOrion UI code:
````
ClientUI::GetTexture(...)
````